amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -81,6 +92,14 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
     if (isLoading) {
       return <LoadingSpinner/>;
     }
+    if (this.props.statusCode === 500) {
+      return (
+        <div className="container error-label">
+          <Breadcrumb />
+          <label>Something went wrong...</label>
```

Comment:
Feel free to add anything better here if you want. This is an artifact from the early days and you don't have to feel obligated to propagate it.

Reply:
Don't have anything better to put at the moment :)